Ray.jl/src/object_store.jl

```diff
@@ -0,0 +1,30 @@
+"""
+    Ray.put(data) -> ObjectIDAllocated
+
+Stores `data` in the object store and returns an object reference assigned to the value that
+can used to retrieve the data with [`Ray.get`](@ref).
+"""
+function put(data)
+    bytes = Vector{UInt8}()
+    io = IOBuffer(bytes; write=true)
+    serialize(io, data)
+    buffer_ptr = Ptr{Nothing}(pointer(bytes))
+    buffer_size = sizeof(bytes)
+    buffer = rayjll.LocalMemoryBuffer(buffer_ptr, buffer_size, true)
+    return rayjll.put(buffer)
+end
+
+"""
+    Ray.get(object_id::ObjectIDAllocated)
+    Ray.get(object_ids::Union{AbstractVector, Tuple})
+
+Retrieves the data associated with the (collection of) `object_id`(s) from the object store.
+This method is blocking until the data is available in the local object store.
+"""
+function get(oid::rayjll.ObjectIDAllocated)
+    io = IOBuffer(take!(rayjll.get(oid)))
+    return deserialize(io)
+end
+
+get(oids::Union{AbstractVector, Tuple}) = map(get, oids)
+get(x) = x
```

Comment:
I'll call out that ray Python only supposes _lists_ of object refs, not tuples, and doesn't have a fallback method either.

I didn't see a reason not to include these, other than to be consistent with what Python doesn't support.
Supporting common collections of object refs and not erroring over non-objectids seems a no brainer?

Reply:
I'll mention I'm for having the `get(x) = x` fallback method. Having this will be important for writing generic Julia functions which may receive a `ObjectID` or may receive the value directly depending on whether it's called locally or as a remote Ray task.

Reply:
makes sense!